core/serialisation/src/main/java/uk/gov/gchq/gaffer/serialisation/implementation/SerialisationFactory.java

```diff
@@ -23,13 +24,17 @@
 import uk.gov.gchq.gaffer.serialisation.implementation.raw.RawFloatSerialiser;
 import uk.gov.gchq.gaffer.serialisation.implementation.raw.RawIntegerSerialiser;
 import uk.gov.gchq.gaffer.serialisation.implementation.raw.RawLongSerialiser;
+import java.util.List;
 
 /**
  * A <code>SerialisationFactory</code> holds a list of core serialisers and
  * is design to provide compatible serialisers for given object classes.
  */
 public class SerialisationFactory {
-    private static final Serialiser[] SERIALISERS = new Serialiser[]{
+
+    private final List<Serialiser> serialisers;
+    private static final Serialiser LAST_RESORT_FINALISER = new JavaSerialiser();
+    private static final Serialiser[] CORE_SERIALISERS = new Serialiser[]{
```

Comment:
If you made this a List you wouldn't need your serialiserAlreadyinList method. You could just do CORE_SERIALISERS.contains(serialiser)